### PR TITLE
added mailchimp service to client

### PR DESCRIPTION
**Pull request by DavidMarom**

### CODE REVIEW

File: src/services/mailchimp.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import http from '@/services/http';
+
+export const sendWelcomeEmail = async (mail: string | null, name: string | null) => {
```
Comment (noaYarin) [inline src/services/mailchimp.ts]: There is async without await

File: src/app/layout.tsx [hunk 2]
```diff
@@ -34,20 +35,10 @@ export default function RootLayout({
...
-            })
-
-            http.post('sendwelcommail', {
-              mail: res.mail,
-              name: res.name,
-            }
-            ).then((response: any) => {
-              console.log(response)
-            })
+            http.post('users', res).then((response: any) => { console.log(response) })
```
Comment (noaYarin) [inline src/app/layout.tsx]: No need for a catch?

File: src/services/mailchimp.ts (hunk 1 above)
Comment (DavidMarom) [reply src/services/mailchimp.ts]: removed

File: src/app/layout.tsx (hunk 2 above)
Comment (DavidMarom) [reply src/app/layout.tsx]: added

**Comment by DavidMarom** [comment]
@YarinLevi5 I fixed the 2 issues

**Pull request merged**